mysite/templates/Login.html

```diff
@@ -17,14 +17,24 @@
 
 <style>
 
+.login-main {
+  min-height: 76vh;
+  display: grid;
+  place-content: center;
+}
+
 .login-body{
   background-color: rgb(30, 30, 30);
   box-shadow: 13px 4px 19px 7px rgb(19 17 17);
   border-radius: 40px;
   color: white;
-  max-width: 450px;
+  min-width: 450px;
   padding: 15px;
   margin: auto;
+  display: flex;
```

Comment:
Hey this is just work around not a generic solution as this problem persists with sign up page as well. we need to write some css logic to footer itself.
![image](https://github.com/calcont/calcont.in/assets/71604396/b6307b3f-802b-47c2-af27-42ed5bbed67a)